src/olympia/reviewers/forms.py

```diff
@@ -200,7 +200,12 @@ def create_option(self, *args, **kwargs):
         # label_from_instance() on ReasonsChoiceField returns the full object,
         # not a label, this is what makes this work.
         obj = option['label']
-        option['attrs']['data-value'] = obj.canned_response
+        canned_response = (
+            obj.cinder_policy.full_text(obj.canned_response)
+            if obj.cinder_policy
+            else obj.canned_response
+        )
+        option['attrs']['data-value'] = f'- {canned_response}\n'
```

Comment:
Should we have `- ` defined in `full_text`? I see it in `policies.txt` and here.

Reply:
I thought about doing that, but I think it only has its place where we know for sure we're (potentially) building a list of policies. In `full_text()` we technically don't, even though at the moment it is the case everywhere.